librz/core/cmd_api.c

```diff
@@ -1089,6 +1089,87 @@ static char *get_help(RzCmd *cmd, RzCmdDesc *cd, RzCmdParsedArgs *args, bool use
 	return NULL;
 }
 
+
+static bool child_help_json(RzCmd *cmd, RzCmdDesc *cd, PJ* j) {
+	pj_k(j, cd->name);
+	pj_o(j);
+	pj_ks(j, "cmd", cd->name);
+	if (cd->help->args_str) {
+		pj_ks(j, "args", cd->help->args_str);
+	} else {
+		RzStrBuf *sb = rz_strbuf_new(NULL);
+		fill_args(sb, cd);
+		char* args = rz_strbuf_drain(sb);
+		pj_ks(j, "args", args);
+		free (args);
+	}
+	pj_ks(j, "desc", cd->help->summary ? cd->help->summary : "");
+	pj_end(j);
+
+	return true;
+}
+
+static bool argv_modes_get_help_json(RzCmd *cmd, RzCmdDesc *cd, PJ *j) {
+	size_t i;
+	for (i = 0; i < RZ_ARRAY_SIZE(argv_modes); i++) {
+		if (cd->d.argv_modes_data.modes & argv_modes[i].mode) {
+			char *name = rz_str_newf("%s%s", cd->name, argv_modes[i].suffix);
+			char *summary = rz_str_newf("%s%s", cd->help->summary, argv_modes[i].summary_suffix);
+
+			pj_k(j, name);
+			pj_o(j); // {
+			pj_ks(j, "cmd", name);
+			if (cd->help->args_str) {
+				pj_ks(j, "args", cd->help->args_str);
+			} else {
+				RzStrBuf *sb = rz_strbuf_new(NULL);
+				fill_args(sb, cd);
+				char* args = rz_strbuf_drain(sb);
+				pj_ks(j, "args", args);
+				free (args);
+			}
+			pj_ks(j, "desc", summary);
```

Comment:
this should be `summary`. Also, you should add `description` field for the actualy description field.